### PR TITLE
Sso authorization step

### PR DESCRIPTION
**Pull request by MaxArnaut**
Do not merge
Need to test

### CODE REVIEW

**Comment by eldad-frontegg** [comment]
> Do not merge
> Need to test

Tested locally and fixed the API calls

**Review by aviadmizrachi — APPROVED** [review]

File: packages/auth/src/SSO/SSOManageAuthorizationPage/SSOManageAuthorizationPage.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  useEffect(() => {
+    loadSSOAllRoles();
+    loadSSOAuthorizationRoles();
+  }, []);
+
+  if (loading) {
+    return null;
+  }
+
+  const prefixT = 'auth.sso.authorization';
```
Comment (frontegg-david) [inline packages/auth/src/SSO/SSOManageAuthorizationPage/SSOManageAuthorizationPage.tsx:22]: extract Title and Subtutle outside the SSOManageAuthPage component.

File: packages/rest-api/src/auth/index.ts [hunk 2]
```diff
@@ -296,6 +297,24 @@ export async function validateSamlDomain(): Promise<ISamlConfiguration> {
   return Put(`${SSO_SERVICE_URL_V1}/saml/validations/domain`);
 }
 
+/**
+ *  Get social logins providers configurations for vendor
+ * @return array of providers configurations
```
Comment (frontegg-david) [inline packages/rest-api/src/auth/index.ts:302]: fix comments and log message

File: packages/rest-api/src/auth/index.ts [hunk 3]
```diff
@@ -296,6 +297,24 @@ export async function validateSamlDomain(): Promise<ISamlConfiguration> {
...
+  console.debug('getSocialLoginsProviders()');
+  return Get(`${SSO_SERVICE_URL_V1}/saml/configurations/roles/default`);
+}
+
+/**
+ *  Get social logins providers configurations for vendor
+ * @return array of providers configurations
+ */
+export async function updateSamlRoles({ roleIds }: IUpdateSamlRoles): Promise<void> {
+  console.debug('getSocialLoginsProviders()');
```
Comment (frontegg-david) [inline packages/rest-api/src/auth/index.ts:314]: do not copy & paste comments.

**Review by frontegg-david — CHANGES_REQUESTED** [review]

**Pull request merged**